server/src/http/unleash_client.rs

```diff
@@ -224,19 +225,29 @@ pub struct EdgeTokens {
 }
 
 impl UnleashClient {
-    pub fn from_url(server_url: Url, token_header: String, backing_client: Client) -> Self {
+    pub fn from_url(
+        server_url: Url,
+        token_header: String,
+        backing_client: Client,
+        client_meta_information: ClientMetaInformation,
+    ) -> Self {
         Self {
             urls: UnleashUrls::from_base_url(server_url),
             backing_client,
             custom_headers: Default::default(),
             token_header,
+            meta_info: client_meta_information.clone(),
```

Comment:
Do we need the clone? This looks like it takes an owned type

```suggestion
            meta_info: client_meta_information,
```

Reply:
No, we didn't. Fixed.